src/pancam.py

```diff
@@ -0,0 +1,22 @@
+from follower import Follower
+
+follower = Follower()
+
+def seek():
+    #tag_data = follower.detect()
+    turn = follower.turn()
+
+    if follower.detect() == False:
+        print('Detect False')
+        follower._speed = follower.LEADER_MAX_SPEED
+        turn = turn + 35
```

Comment:
This variable doesn't seem to be used anywhere in the code. Turn what 35 and in which direction? Why is it 35 in the first place?